### PR TITLE
update link cpp training

### PR DESCRIPTION
**Pull request by klieret**
- Linking to links in readme rather than outdated
- Drop and move modules
- Make scikit-HEP stable

### CODE REVIEW

**Comment by klieret** [comment]
Jim has informed me that we can move `uproot` to the archived section.

File: _training/curriculum.md [hunk 1]
```diff
@@ -21,23 +21,23 @@ The curriculum is comprised of a set of standardized *modules*, so that students
 
 ### C++ corner
 
-{% include list_of_selected_training_modules.html ids="hepcpp,moderncpp,cmake"%}
+{% include list_of_selected_training_modules.html ids="hepcpp,cmake"%}
```
Comment (wdconinc) [inline _training/curriculum.md:24]: Should we add moderncpp to archived too then?

Comment (klieret) [reply _training/curriculum.md:24]: 👍  Ah sure. Though there is really not much to be seen there.

**Review by wdconinc — APPROVED** [review]

**Comment by klieret** [comment]
Thanks!

**Pull request merged**